docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Encrypt and decrypt exported and imported files
+description: Encrypt and decrypt exported and imported files
+layout: ../../../layouts/MainLayout.astro
+---
+
+# Spike -- Encrypt & Decrypt configuration
+
+## Document link
+
+Due to confidentiality reasons, spike documents are only accessible to internal personnel.
+
+[Encrypt & Decrypt configuration](https://docs.google.com/document/d/1Drdi8sXX37QdKFdOJB3WgFfjrBNRfNTH/edit)
```

Comment:
Please `as doc` here, not link to other website

Reply:
hello，zhongren，Due to the encryption and decryption process being only visible to internal personnel, so only the Google Drive link is placed in the docs

Reply:
Hey, @BoBoDai 
as an open-source tool, should keep every thing open.
So, Covert the google doc as code here.
It is very important to store the `diagram as code` here

Reply:
Okay, I may need to ask the security team first, and then we can discuss it.

Reply:
Hi @BoBoDai , after discussed with someone from security, I think @guzhongren is correct. Keep it open and put the document here is okay. There are a lot of open source encryptions outside. It doesn't mean they are insecure. So keep it open should be a good way.

Reply:
ok,i got it